opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -270,12 +270,14 @@ class ConsoleSpanExporter(SpanExporter):
     def __init__(
         self,
         out: typing.IO = sys.stdout,
-        formatter: typing.Callable[[Span], str] = str,
+        formatter: typing.Callable[[Span], str] = lambda span: str(span)
+        + "\n",
     ):
         self.out = out
         self.formatter = formatter
 
     def export(self, spans: typing.Sequence[Span]) -> SpanExportResult:
         for span in spans:
             self.out.write(self.formatter(span))
```

Comment:
Consider doing this to avoid two linebreaks for formatters that do end in a newline:

```suggestion
            self.out.write(endline(self.formatter(span)))
```

with this in the module:

```python
def endline(line):
    if line.endswith(os.linesep):
        return line
    return line + os.linesep
```

I haven't checked for windows, but suspect you do want `os.linesep` here instead of `"\n"`.

Reply:
I'm not totally sure about this one, I think we should let full control to the user, for instance, there could be a case where the user doesn't want to have line breaks at all and use another separator.

I totally agree about the `os.linesep`, I'm too focus on Linux now :stuck_out_tongue_closed_eyes:

Reply:
Btw, if a formatter ends with a linebreak there would not be two linebreaks, I only adding a linebreak to the default formatter.

Reply:
I think agreed to not force lineseps, formatter should control that (and the flush handles this anyway).